CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `opentelemetry-instrumentation-grpc` add supports to filter requests to instrument. ([#1241](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1241))
 - Flask sqlalchemy psycopg2 integration
   ([#1224](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1224))
+- `opentelemetry-instrumentation-asgi` metrics record target attribute (FastAPI only)
```

Comment:
move this to unrelated section

Reply:
What do you mean by this? I did not found any unrelated sections?

The entry is already in the `Unreleased` section.

Reply:
Ah sorry, that was a typo. 

> The entry is already in the Unreleased section.

Not if you resolve the conflicts. Please resolve conflicts and move to the unreleased section.

Reply:
@srikanthccv updated, please take another look